modules/ags/src/main/scala/lucuma/ags/GuideProbe.scala

```diff
@@ -9,6 +9,4 @@ import lucuma.core.util.Enumerated
  * Enumerated type for guide probe
  */
 enum GuideProbe(private val tag: String) derives Enumerated:
-  case AOWFS extends GuideProbe("AOWFS")
-  case OIWFS extends GuideProbe("OIWFS")
-  case PWFS  extends GuideProbe("PWFS")
+  case GmosOiwfs extends GuideProbe("GmosOiwfs")
```

Comment:
I didn't know any of the specifics for `AOWFS` or `PWFS` so I removed them for now. 
Note that the `toString` of the enum is used in the `message` methods of the `AgsAnalysis` types. Is the new name OK for the messages, or do we want to make something better for it. We could `screaming snake case` it to match the graphql enums, add a display value, or...